utm.yaml

```diff
@@ -1122,12 +1122,17 @@ components:
             $ref: '#/components/schemas/Volume4D'
         state:
           $ref: '#/components/schemas/OperationState'
-        vlos:
-          description: If true, this Operation is ensuring deconfliction visually
-            and volumes of other Operations may overlap the volumes of this Operation.
-          type: boolean
-          default: false
+        priority:
+          $ref: '#/components/schemas/Priority'
 
+    Priority:
+      description: |-
+        Ordinal priority of the Operation, as defined by the regulator.  A lower-priority Operation
+        may not have any conflicts with a higher-priority Operation.  A higher-priority Operation
```

Comment:
Reword to "lower-priority operation may not create a conflict with a higher-priority Operation." Since higher-priority Operations can overwrite, the lower-priority operation would "have" a conflict for some period of time.

Reply:
Done.

Reply:
Ok. @BenjaminPelletier please resolve (I can't since I don't have write access).